_episodes/02-setup.md

```diff
@@ -52,8 +52,8 @@ For this lesson, we will be interacting with [GitHub](https://github.com/) and s
 > Different operating systems use different character(s) to represent the end of a line.
 > (You may also hear these referred to as newlines or line breaks.)
 > Because Git uses these characters to compare files,
-> it may cause unexpected issues when editing a file on different machines. 
-> Though it is beyond the scope of this lesson, you can read more about this issue 
+> it may cause unexpected issues when editing a file on different machines.
+> Though it is beyond the scope of this lesson, you can read more about this issue
 > [in the Pro Git book](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf).
```

Comment:
This link may be the problem. The `#_core_autocrlf` should be `#core_autocrlf`. However, the anchor seems to be. missing and the link goes to the top of the page. Maybe use this instead:

```
you can read more about this issue under `core.autocrlf` in the [Pro Git book](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration).
```